misc/alert_rules.json

```diff
@@ -438,24 +438,24 @@
  {
     "rule": "customoids.customoid_current >= `customoids.customoid_limit` && customoids.customoid_alert = \"1\" && macros.device_up = \"1\"",
     "name": "CustomOID over limit",
-    "default": true
```

Comment:
Isn't it enough to simply remove the key completely to disable it?

Reply:
sure